cargo-workspaces/src/utils/mod.rs

```diff
@@ -7,8 +8,10 @@ mod error;
 mod git;
 mod listable;
 mod pkg;
+pub(crate) mod publish_helpers;
```

Comment:
```suggestion
mod publish_helpers;
```

and use `pub use` below.